cbw_api_toolbox/cbw_api.py

```diff
@@ -167,3 +169,42 @@ def update_remote_access(self, remote_access_id, info):
 
         logging.error("Error update remote access")
         return False
+
+    def import_remote_accesses_xlsx(self, file_xlsx):
+        """method to import remote accesses from an xlsx file"""
+        if file_xlsx:
+            file = file_xlsx.split(".")
+            if file[len(file) - 1] == "xlsx": #check if the extension is xlsx
+                imported = xlrd.open_workbook(file_xlsx)
+                lines = imported.sheet_by_name(u'Example')
+                response = []
+
+                for line in range(1, lines.nrows):
+                    address = lines.row_values(line)[0]
+                    type = lines.row_values(line)[3] # pylint: disable=redefined-builtin
+                    auth = lines.row_values(line)[5]
+                    logging.debug("Creating remote access {}".format(address))
+                    info = {
+                        "address": lines.row_values(line)[0],
+                        "port": lines.row_values(line)[1],
+                        "type": REMOTE_ACCES_TYPE[lines.row_values(line)[2]][type],
+                        "login": lines.row_values(line)[4],
+                        "node": lines.row_values(line)[6],
+                        "password": auth if type == "PASSWORD" else "",
+                        "key": auth if type == "KEY" else ""
+                    }
+
+                    response.append(self.create_remote_access(info))
+                    logging.debug("Remote access {} successfully created".format(address))
+                    remote_accesses = self.remote_accesses()
```

Comment:
It's a shame to browse all the remote accesses while the server id is in the create_remote_access response